cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -189,3 +210,15 @@ mkTopologyConfig numNodes allPorts port True = Aeson.encode topologyP2P
         localRootPeerGroups
         []
         (UseLedger DontUseLedger)
+
+
+convertToEraString :: AnyCardanoEra -> String
+convertToEraString (AnyCardanoEra e) =
+  case e of
+    ConwayEra -> "conway"
+    BabbageEra -> "babbage"
+    AlonzoEra -> "alonzo"
+    MaryEra -> "mary"
+    AllegraEra -> "allegra"
+    ShelleyEra -> "shelley"
+    ByronEra -> "byron"
```

Comment:
You can use this instead:
```suggestion
convertToEraString = map toLower . docToString . pshow
```

Reply:
This actually gives "conwayera". I'm reverting to what I originally had.